src/Compilers/Core/Portable/Serialization/FixedObjectBinder.cs

```diff
@@ -1,48 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Immutable;
-
-namespace Roslyn.Utilities
-{
-    /// <summary>
-    /// An <see cref="ObjectBinder"/> with a fixed set of type and reader mappings.
-    /// </summary>
-    internal class FixedObjectBinder : ObjectBinder
```

Comment:
This allowed us to have no virtualization and no need to have any sort of indirection between the base and derived types.